modules/programs/atuin.nix

```diff
@@ -129,7 +129,9 @@ in {
         if not ($atuin_cache | path exists) {
           mkdir $atuin_cache
         }
-        ${cfg.package}/bin/atuin init nu | save --force ${config.xdg.cacheHome}/atuin/init.nu
+        ${cfg.package}/bin/atuin init nu ${
+          builtins.concatStringsSep " " cfg.flags
```

Comment:
```suggestion
          flagsStr
```

Reply:
Ah, I thought I tested that and found it did not work. Just tested it again, and I guess I did something wrong initially.